util/__init__.py

```diff
@@ -1,7 +1,11 @@
 from __future__ import absolute_import, division, print_function
 
+__all__ = ["progress"]
+
 import sys
 
+from ._progress import progress
```

Comment:
do we have to?

I know ```from dials.util.progress import progress``` doesn't look as nice

Reply:
I suppose it could be argued that unnecessary repetition is totally idiomatic of our codebase, things like:
```
from dxtbx.format.Format import Format
from dxtbx.datablock import Datablock
```
for which at least `Format` has the excuse that it's the same case as the module. But it really doesn't look nice, and package layout should be an implementation detail, right?